public_dropin_environments/java_h2o/README.md

```diff
@@ -0,0 +1,20 @@
+# Java Drop-In Template Environment
+
+This template can be used as an environment for DataRobot generated scoring code or models that implement the either the `IClassificationPredictor`
+or `IRegressionPredictor` interface from the [datarobot-prediction](https://mvnrepository.com/artifact/com.datarobot/datarobot-prediction) package.
+
+## Requirements
+
+- Java 11 JDK
+- A valid scoring code jar
```

Comment:
what about a java file? Could you mention about where to obtain these?

Reply:
updating readmes